geomesa-accumulo/geomesa-accumulo-dist/src/main/assemblies/component.xml

```diff
@@ -60,7 +60,7 @@
             <useProjectArtifact>false</useProjectArtifact>
             <useTransitiveDependencies>false</useTransitiveDependencies>
             <includes>
-                <include>org.locationtech.geomesa:geomesa-accumulo-compute*</include>
+                <include>org.locationtech.geomesa:geomesa-spark*</include>
```

Comment:
we probably don't want to bundle the generic data store one here - not sure about all others